apps/portal/src/content/docs/components/logo.mdx

```diff
@@ -2,72 +2,30 @@
 title: Logo
 description: Logo component
 beta: true
-draft: true
 ---
 
 The `Logo` component renders brand icons from the `simple-icons` package.
```

Comment:
I don't think we are directly using `simple-icons` anymore, so this line is incorrect. let's remove it.

Reply:
Sure. Portal will be updated entirely 👍